src/docs/development/platform-integration/add-to-app-android/add-splash-screen.md

```diff
@@ -0,0 +1,221 @@
+---
+title: Adding a splash screen and launch screen to an Android app
```

Comment:
Most of this page is more relevant for the full-app case (once we make the FullFlutterApp which I assume will have the same functionality) such as https://flutter.dev/docs/development/ui/assets-and-images#updating-the-launch-screen than add to app. 

Can we update that section so full-app developers can find instructions too and for the add to app specific things such as FlutterFragment, we can leave them here or let them be in the Fragment guide?

Reply:
I replaced the description in the guide you linked with a cross-reference to this new guide, which explains the same details and much more.